docs/help/index.md

```diff
@@ -1,43 +1,31 @@
 ---
 layout: default
-title: Help Intro
+title: Getting Started
 lang: en
 ref: intro
 ---
 
-Help Intro
+Getting Started
 ==========
-+ [Experimenting with bobbin lace grounds](#experimenting-with-bobbin-lace-grounds)
 + [Virtual tour](#virtual-tour)
-+ [Overview Tutorial](#overview-tutorial)
++ [GroundForge 101](#groundforge-101)
    + [Highlight a thread](#highlight-a-thread)
    + [Resize pattern](#resize-pattern)
    + [Modify stitches](#modify-stitches)
    + [Undo changes](#undo-changes)
    + [Save and share changes](#save-and-share-changes)
    + [Create a new ground pattern](#create-a-new-ground-pattern)
    + [Add footsides](#add-footsides)
-   + [Binche snowflakes](#next-step)
+   + [Next steps](#next-steps)
    
 
-Experimenting with bobbin lace grounds
---------------------------------------
-
-GroundForge is a tool that complements existing lace ground catalogues ([TesseLace](/tesselace-to-gf/), [Whiting](/gw-lace-to-gf/), [Flanders](https://maetempels.github.io/MAE-gf/docs/flanders), [MAE](https://maetempels.github.io/MAE-gf/)) by making it possible to play with their designs.
-GroundForge generates an interactive thread diagram for each ground in the catalogue which allows you to highlight the path of a thread; a useful feature when working with multiple thread colours or thicknesses in a design.
-The tool also allows you to specify the combinations of stitches that are made when two pairs meet.  You can make changes and quickly observe how this affects the thread diagram.
-This greatly speeds up the process of experimenting with lace grounds; no need to wind bobbins or prepare prickings. The final test is always to make a hand-worked sample&mdash;only in a physical sample can the interactions between pins and threads under tension be completely observed&mdash;however, GroundForge can help you narrow the number of samples you need to test by hand.
-You can experiment and play around with the hundreds of patterns provided here, or you can create your own.
-Even when reinventing the wheel, your own discoveries can bring great joy.
-
-
 Virtual tour
 ------------
 
 Take a walk through the various catalogues on the GroundForge website in this [screencast](https://github.com/d-bl/GroundForge/releases/download/2019-Q2/catalogues.mp4) (MP4 format, 123 MB, 2:32 min). You will notice that Rose ground, and many variations related to it, appears in each of the catalogues.  
```

Comment:
The two spaces at the end of this line cause a line break in the decorated version of the page. Is that intentional?

Reply:
Not intentional.